docs/maintainer-guide/releases.md

```diff
@@ -1,29 +1,29 @@
 # Performing releases
 
 1. Create a [new issue](https://github.com/stylelint/stylelint/issues/new) announcing the planned release, e.g. `Release 8.11.1` and include the [template checklist](#new-release-issue-template).
-2. Locally test `main` in the `stylelint-config-*` shareable config repositories. Install current `main` branch (`npm install stylelint/stylelint#main`) and run tests.
-3. Locally test `main` in the [stylelint/stylelint.io](https://github.com/stylelint/stylelint.io) repository.
-4. Locally test `main` in the [stylelint/stylelint-demo](https://github.com/stylelint/stylelint-demo) repository.
+2. Locally test `main` in the `stylelint-config-*` shareable config repositories. Install the current `main` branch (`npm install stylelint/stylelint#main --save-dev`) and run tests.
+3. Locally test `main` in the [stylelint/stylelint-demo](https://github.com/stylelint/stylelint-demo) repository.
+4. Locally test `main` in the [stylelint/stylelint.io](https://github.com/stylelint/stylelint.io) repository.
 5. Publish the package to npm and create a GitHub release using [`np`](https://github.com/sindresorhus/np):
    1. [Consistently format](pull-requests.md) the [changelog](../../CHANGELOG.md).
    2. Replace `## Head` with new version number e.g. `## 8.1.2`.
    3. Commit these changes.
    4. Push these changes.
-   5. Confirm the changes are correct at [https://github.com/stylelint/stylelint](https://github.com/stylelint/stylelint).
+   5. Confirm the changes are correct at [github.com/stylelint/stylelint](https://github.com/stylelint/stylelint).
    6. Run `npm run release`.
    7. Select the version that matches the one from the changelog.
    8. Copy and paste the changelog entries for the published version from [changelog](../../CHANGELOG.md) when the GitHub release page opens.
-   9. Confirm the publishing of the package to [https://www.npmjs.com/package/stylelint](https://www.npmjs.com/package/stylelint).
-   10. Confirm the creation of the release at [https://github.com/stylelint/stylelint/releases](https://github.com/stylelint/stylelint/releases).
+   9. Confirm the publishing of the package to [www.npmjs.com/package/stylelint](https://www.npmjs.com/package/stylelint).
+   10. Confirm the creation of the release at [github.com/stylelint/stylelint/releases](https://github.com/stylelint/stylelint/releases).
```

Comment:
[note] For consistency and simplicity, I've omitted the `https://` protocol from the links.